.rubocop.yml

```diff
@@ -39,3 +39,12 @@ RSpec/SpecFilePathFormat:
 
 RSpec/SpecFilePathSuffix:
   Enabled: true
+
+RSpec/RepeatedExample:
+  Exclude:
+    - spec/policies/*.rb
+
+RSpec/RepeatedDescription:
+  Exclude:
+    - spec/policies/*.rb
+
```

Comment:
Testing policies with these rules is frustrating

Reply:
This change shouldn't be necessary if you wrap the test in context blocks:
```ruby
permissions :edit? do
  context "when user has an internal claim" do
    it "grants access" do
      expect(claim_policy).to permit(user, internal_claim)
    end
  end
  
  context "when the user has a draft claim" do
    it "grants access" do
      expect(claim_policy).to permit(user, draft_claim)
    end
  end

  context "when the user has a submitted claim" do
    it "denies access" do
      expect(claim_policy).not_to permit(user, submitted_claim)
    end
  end
end
```